### PR TITLE
[Artur Kostiuk]uit 64 swatch selection

### PR DESCRIPTION
**Pull request by art1kos**

### CODE REVIEW

File: src/app/product-details-page/components/product-description/product-description.component.html [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+        <div class="details-container-info">
+          <h2>{{product.name}}</h2>
+          <h5>{{product.title}}</h5>
+          <p>{{product.description}}</p>
+        </div>
 
-      <div class="details-container-order">
-        <app-product-order [product]="product"></app-product-order>
+        <div class="details-container-order">
+          <app-product-order (selectedSwatchImg)="imgChanger($event)" [product]="product"></app-product-order>
```
Comment (dverbovyi) [inline src/app/product-details-page/components/product-description/product-description.component.html]: imgChanger?))))) xD please rename

File: src/app/product-details-page/components/product-description/product-description.component.ts [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
 @Component({
   selector: 'app-product-description',
   templateUrl: './product-description.component.html',
   styleUrls: ['./product-description.component.scss']
 })
 export class ProductDescriptionComponent {
   @Input() product: IProduct;
+  selectedColorFromOrder;
+
+  imgChanger(e) {
```
Comment (dverbovyi) [inline src/app/product-details-page/components/product-description/product-description.component.ts]: onSwatchSelect

File: src/app/product-details-page/components/product-description/product-description.component.html (hunk 1 above)
Comment (dverbovyi) [inline src/app/product-details-page/components/product-description/product-description.component.html]: (swatchSelect)="onSwatchSelect"

File: src/app/product-details-page/components/product-description/product-image-preview/product-image-preview.component.ts [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
   templateUrl: './product-image-preview.component.html',
   styleUrls: ['./product-image-preview.component.scss']
 })
-export class ProductImagePreviewComponent implements OnChanges {
+
+export class ProductImagePreviewComponent implements OnInit, OnChanges {
   @Input() thumbnailImageSrc: string;
-  @Input() swatches = [];
+  @Input() swatches: any[];
+  @Input() selectedColorFromOrder;
```
Comment (dverbovyi) [inline src/app/product-details-page/components/product-description/product-image-preview/product-image-preview.component.ts]: selectedSwatch

**Review by dverbovyi — CHANGES_REQUESTED** [review]
@art1kos good, please fix minor comments

**Comment by art1kos** [comment]
https://issues.griddynamics.net/browse/UIT-64

**Review by dverbovyi — APPROVED** [review]

**Pull request merged**